build.gradle

```diff
@@ -91,13 +91,13 @@ repositories {
 
 dependencies {
 	compile 'com.dell.isg.smi:wsmanlib:1.0.29'
-	compile 'com.dell.isg.smi:wsmanclient:1.0.28'
+	compile 'com.dell.isg.smi:wsmanclient:1.0.17'
```

Comment:
Why are you changing back to version 17?  is there a problem with 28?  version 28 is published on maven central.
http://repo1.maven.org/maven2/com/dell/isg/smi/wsmanclient/